src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -5566,6 +5568,247 @@ void GetVRFInputData(EnergyPlusData &state, bool &ErrorsFound)
     }
 }
 
+void CheckVRFTUNodeConnections(EnergyPlusData &state, int const VRFTUNum, bool &ErrorsFound)
+{
+
+    std::string const cTerminalUnitType("ZoneHVAC:TerminalUnit:VariableRefrigerantFlow");
```

Comment:
Checking with @Myoldmopar about our current best practice for declaring string constants. This is nit-picking for a one-time function, but I'd like to make sure new code doesn't perpetuate old patterns.

Reply:
@mjwitte I found where I had seen @lefticus document the best practice.  The message is:

> For any future update of const static std::string, I recommend a move to constexpr static std::string_view and updating all downstream code to avoid as many dynamic allocations as possible.

For a fuller discussion, check the main comment body of #8663.  I now have a better feel for how to utilize string views; this is a really exciting change with lots of potential in our code.

Reply:
So constexpr static std::string_view cTerminalUnitType("string"); should be used here?